src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -3263,14 +3316,15 @@ void func_80836448(PlayState* play, Player* this, LinkAnimationHeader* anim) {
     }
 }
 
-s32 func_808365C8(Player* this) {
+s32 Player_CanUpdateItems(Player* this) {
     return (!(Player_Action_808458D0 == this->actionFunc) ||
-            ((this->stateFlags1 & PLAYER_STATE1_8) &&
+            ((this->stateFlags1 & PLAYER_STATE1_START_CHANGING_HELD_ITEM) &&
              ((this->heldItemId == ITEM_LAST_USED) || (this->heldItemId == ITEM_NONE)))) &&
-           (!(func_80834A2C == this->func_82C) || (Player_ItemToItemAction(this->heldItemId) == this->heldItemAction));
+           (!(Player_IA_ChangeHeldItem == this->itemActionFunc) ||
```

Comment:
```suggestion
           ((Player_IA_ChangeHeldItem != this->itemActionFunc) ||
```
Does this match?

Reply:
doesnt seem to 
![image](https://github.com/zeldaret/oot/assets/22045047/213bdc8e-ac9c-4e99-93f2-7dab9b48a85c)